templates/devtest/gitea-ui.tmpl

```diff
@@ -14,24 +14,71 @@
+				.button-sample-groups .sample-group .ui.button { margin-bottom: 5px; }
+			</style>
+			<ul class="button-sample-groups">
+				<li class="sample-group">
+					<h2>General purpose:</h2>
+					<button class="ui button">Unclassed</button>
+					<button class="ui basic button">Basic Unclassed</button>
+					<button class="ui primary button">Primary</button>
+					<button class="ui basic primary button">Basic Primary</button>
+					<button class="ui negative button">Negative</button>
+					<button class="ui basic negative button">Basic Negative</button>
+					<button class="ui positive button">Positive</button>
+					<button class="ui basic positive button">Basic Positive</button>
+				</li>
+				<li class="sample-group">
+					<h2>Recommended colors:</h2>
+					<button class="ui red button">Red</button>
+					<button class="ui basic red button">Basic Red</button>
+					<button class="ui green button">Green</button>
+					<button class="ui basic green button">Basic Green</button>
+					<button class="ui blue button">Blue</button>
+					<button class="ui basic blue button">Basic Blue</button>
+					<button class="ui orange button">Orange</button>
+					<button class="ui basic orange button">Basic Orange</button>
+					<button class="ui yellow button">Yellow</button>
+					<button class="ui basic yellow button">Basic Yellow</button>
+				</li>
+				<li class="sample-group">
+					<h2>Supported but not recommended:</h2>
+					<p>Do not use if there is no strong requirement. Do not use grey/black buttons, they don't work well with dark theme.</p>
+					<button class="ui secondary button">Secondary</button>
+					<button class="ui basic secondary button">Basic Secondary</button>
+					<button class="ui olive button">Olive</button>
+					<button class="ui basic olive button">Basic Olive</button>
+					<button class="ui teal button">Teal</button>
+					<button class="ui basic teal button">Basic Teal</button>
+					<button class="ui violet button">Violet</button>
+					<button class="ui basic violet button">Basic Violet</button>
+					<button class="ui purple button">Purple</button>
+					<button class="ui basic purple button">Basic Purple</button>
+					<button class="ui pink button">Pink</button>
+					<button class="ui basic pink button">Basic Pink</button>
+					<button class="ui brown button">Brown</button>
+					<button class="ui basic brown button">Basic Brown</button>
+				</li>
+				<li class="sample-group">
+					<h2>Inline / Plain:</h2>
+					<div class="gt-my-2">This is a <button class="button button-ghost gt-p-3">Ghost button</button> with padding, hover background, no text highlight</div>
+					<div class="gt-my-2">This is a <button class="button button-link">Link button</button> with no background, hover text highlight + underline</div>
+					<div class="gt-my-2">This is a <button class="button button-link gt-no-underline">No Underline Link button</button> with no background, hover text highlight
```

Comment:
As a "button", I think in most cases (or in all cases) we do not want this underline.

Do you have a case where we need to use a button with hover underline?

Reply:
IMO an element with underline (aka link) means it navigates, but "repo topic edit" is not for this case, so that's why I didn't consider it as a "link button".

From my understanding I still can not think about a real case for using "link button" (aka hover underline) in Gitea.

That's just my opinion, no block from my side if you think it is right.

Reply:
I agree it's not ideal, but it was like that in 1.19 where it was `<a id="manage_topic" class="muted">Manage Topics</a>`. I plan to refactor the topic edit later so that this button becomes a real button  besides the search box, but I think it's not in scope for this PR.

Reply:
> Does Gitea really need the "link button", is there still a case where it needs to use a button with hover underline? If no: is button-link gt-no-underline the right thing to be used everywhere, is something like button-plain (or other single class name) clearer and better than it?

Hmm I need to check the existing `button-link` cases. Certainly does make sense to have stuff that does not navigate not underline.